components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -144,22 +163,18 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
 
     const getRightComponent = (): JSX.Element | undefined => {
         if (rightComponent) {
-            return (
-                <div style={{ flex: '0 0 auto', marginLeft: 16, display: 'flex', alignItems: 'center' }}>
-                    {rightComponent}
-                </div>
-            );
+            return <div className={clsx(defaultClasses.rightComponent, classes.rightComponent)}>{rightComponent}</div>;
```

Comment:
do we want to use the same class (rightComponent) for chevron and rightComponent since chevron is just a shortcut for rightcomponent?

Reply:
rightComponent has different styles applied to it than chevron does; I think it would add unnecessary margins if we applied the rightComponent class.